content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -278,6 +278,7 @@ The Kubernetes project suggests that you use the [AWS EBS](https://github.com/ku
 out-of-tree storage driver instead.
 
 Here is an example StorageClass for the AWS EBS CSI driver:
+
 ```yaml
```

Comment:
Could we use a `code_sample` shortcode here? See https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#source-code-files

Reply:
Good idea